test/performance/scheduler/default_rangespec.yaml

```diff
@@ -8,12 +8,6 @@ cmd:
   # Average value 351116.4 (+/- 0.9%), setting at +20%
   maxWallMs: 425_000
 
-  # Average value 396 mCPU (+/- 8%), setting at +35%
-  mCPU: 535
-  
-  # Average value 445012 (+/- 0.3%), setting at +5%
-  maxrss: 468_000
```

Comment:
Should we also delete from struct?

https://github.com/kubernetes-sigs/kueue/blob/a6adf86c53e4e4e237a280949a4189056f5ed6eb/test/performance/scheduler/checker/checker_test.go#L39-L40